query-algebrizer/src/clauses/convert.rs

```diff
@@ -27,10 +29,7 @@ use clauses::{
     ConjoiningClauses,
 };
 
-use errors::{
-    ErrorKind,
-    Result,
-};
+use errors::AlgebrizerError;
```

Comment:
I'm not against declaring `Result<T> = std::result::Result<T, errors::AlgebrizerError>` to reduce the changes.  Or even implementing `bail!` in each crate -- it's easy.

The `Err(...)?` pattern is odd to me -- I prefer the `return Err(...)` expression.

Reply:
Using `?` adds an implicit `into` to do conversion of errors, which is a little more compact than `return Err(some_err.into())` or `return some_result.map_err(|e| e.into())`.